flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "A flake for s2n-tls";
+
+  inputs.nixpkgs.url = github:NixOS/nixpkgs/nixos-22.11;
+
+  outputs = { self, nix, nixpkgs, flake-utils }: 
+     flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system};
+   in rec {
```

Comment:
Nit: indention

Reply:
would be good to apply some automatic formatting here: https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-fmt.html